### DISCUSSION TITLE
chatGPT的override好像没有用，我按照wiki填davinci的override选项但是一直格式错误

### DISCUSSION THREAD

**Discussion by realzuoc** [Tsuk1ko/CQ-picfinder-robot#403] [问题求助] [answered]
代码小白一枚，
问下大大是wiki里面那个样式有问题吗，我没法编译这个
"overrides": [
      {
        "regexp": "^chatgptcode([\\s\\S]+)",
        "model": "code-davinci-002",
        "useChatAPI": false,
        "maxTokens": 3072,
      }
    ]

**Comment by Tsuk1ko** [comment, marked as answer]
方括号后面是不是没补逗号……

你先学学json格式

**Reply by realzuoc** [reply]
草..................我的，确实没学过json

**Reply by realzuoc** [reply]
发现应该是maxTokens那边结尾多了个逗号，，问题解决！
